src/Order/Customer/Locale.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace GingerPayments\Payment\Order\Customer;
+
+use GingerPayments\Payment\Common\StringBasedValueObject;
+use Assert\Assertion as Guard;
+
+final class Locale
+{
+    use StringBasedValueObject;
+
+    /**
+     * @param string $value
+     */
+    private function __construct($value)
+    {
+        if (!empty($value)) {
+            Guard::regex($value, '/^[a-zA-Z]{2}(_[a-zA-Z]{2})?$/', "Locale : ".$value);
```

Comment:
You could improve the validator regex a little, since the format is aa_BB. Maybe something like this: `/^[a-z]{2}(_[A-Z]{2})?$/`.